eng/common/CHANGELOG.md

```diff
@@ -0,0 +1,35 @@
+This document contains a record of commits to the common engineering system that may be backwards incompatible with older versions of the codebase. In cases where SDK code must be released and/or pipelines must pass for historical commits (e.g. a backport to a hotfix branch), the following commits may need to be applied.
+
+The commit SHAs below are listed based on the commit in the [azure-sdk-tools](https://github.com/Azure/azure-sdk-tools) repository. These changes are then synced to the SDK repositories. To backport one of these commits to an SDK repository, run the following:
+
+```
+# Run this if using SSH auth
+git remote add tools git@github.com:azure/azure-sdk-tools.git
```

Comment:
As we chatted I don't think setting up a remote across the tools and language repos to try and cherry-pick changes will work very well.